src/utils/source.js

```diff
@@ -59,8 +60,17 @@ export function shouldPrettyPrint(source: Source) {
   const _isJavaScript = isJavaScript(source);
   const isOriginal = isOriginalId(source.id);
   const hasSourceMap = source.sourceMapURL;
+  const sourceMapsEnabled = prefs.clientSourceMapsEnabled;
+
+  const options = [
+    _isPretty,
+    !_isJavaScript,
+    isOriginal,
+    hasSourceMap,
+    sourceMapsEnabled
+  ];
 
-  if (_isPretty || isOriginal || hasSourceMap || !_isJavaScript) {
+  if (options.some(Boolean)) {
```

Comment:
I think I'd prefer something a bit more straight forward:

```js
if (
  isJavaScript(source) ||
  isOriginalId(source.id)
) {
  // stuff
} 
```

Reply:
@jasonLaster Is it safe to ignore `isPretty(source)`?